static_src/stores/activity_store.js

```diff
@@ -60,21 +60,33 @@ class ActivityStore extends BaseStore {
     super();
     this._data = new Immutable.List();
     this.subscribe(() => this._registerToActions.bind(this));
+    this.eventsFetched = false;
+    this.eventsFetching = false;
+    this.logsFetched = false;
+    this.logsFetching = false;
+  }
+
+  get fetched() {
```

Comment:
I really like this! By making the store interface include these values, we can enable any of the stores to calculate them on their own.